### ISSUE TITLE
Move manifest checker to Travis-CI

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8632]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#8632|
|Type|enhancement|
|Created|2016-07-20 06:31:55Z|
|Branch|https://github.com/twisted/twisted/tree/8632-travis-manifest|

so that we reduce the number of things which depend on buildbot ... and allow non-team member to have this test executed for their PR

<details><summary>Searchable metadata</summary>

```
trac-id__8632 8632
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__None None
branch__8632_travis_manifest 8632-travis-manifest
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1468996315394888 1468996315394888
changetime__1469018666869544 1469018666869544
version__None None
owner__adiroiban adiroiban

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban removed owner|
|-|-|

PR at https://github.com/twisted/twisted/pull/360 

please review

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc set owner to @adiroiban|
|-|-|

Merge conflicts need to be resolved, now that major changes went into .travis.yml

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban removed owner|
|-|-|

Thanks for the review.

I have updated the branch to use the new static checkers travis job

Please check the changes.

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_4"></a>|@rodrigc set owner to @adiroiban|
|-|-|

https://github.com/twisted/twisted/pull/360#issuecomment-233937077

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub set status to `closed`|
|-|-|

In changeset a2b5087c02c7d245070aff27ec24e2bf6da4aef9
```
#!CommitTicketReference repository="" revision="a2b5087c02c7d245070aff27ec24e2bf6da4aef9"
Merge 8632-travis-manifest: Add manifest checker to Travis-CI.


Author: adiroiban
Reviewer: rodrigc
Fixes: #8632
```

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_6"></a>|@adiroiban commented|
|-|-|

Thanks for the review. Will remove the checker from buildbot.
